lib/hu_board/services/slack.rb

```diff
@@ -29,7 +29,7 @@ def github_url
     def transform_moved(mash)
       {
         username: "#{mash.meta.user.login} via HuBoard",
-        icon_url: "https://avatars.githubusercontent.com/u/#{mash.meta.user.id}?s=64",
+        icon_url: mash.payload.issue.user.avatar_url,
```

Comment:
Didn't notice this before... Why `payload.issue.user` instead of `meta.user.login`? I would think they should match?